iscsi/iscsi.go

```diff
@@ -46,6 +46,8 @@ type Connector struct {
 	Multipath        bool     `json:"multipath"`
 	RetryCount       int32    `json:"retry_count"`
 	CheckInterval    int32    `json:"check_interval"`
+	Discovery        bool     `json:"discovery"`
```

Comment:
Maybe "DoDiscovery" and "DoCHAPDiscovery" would be more descriptive here?

Reply:
Sounds good